codegen/gateway.go

```diff
@@ -564,12 +563,31 @@ func NewEndpointSpec(
 	return augmentHTTPEndpointSpec(espec, endpointConfigObj, midSpecs)
 }
 
+func testFixtures(endpointConfigObj map[string]interface{}) (ret map[string]*EndpointTestFixture, rErr error) {
```

Comment:
Don't use named returned variables for the sake of avoiding some declarations inside the func. 

The main problem of named return is that it is not clear about the return value at the line of return. Readers have to go back to the previous code to find out the value of each variable.

https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters